api/rest/restapi.go

```diff
@@ -654,6 +666,37 @@ func (api *API) unpinHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (api *API) pinPathHandler(w http.ResponseWriter, r *http.Request) {
+	var ci cid.Cid
+	pinPath, ps := api.parsePathOrError(w, r)
+	logger.Debugf("rest api pinPathHandler: %s", pinPath)
+	err := api.rpcClient.Call(
+		"",
+		"Cluster",
+		"PinPath",
+		[]interface{}{pinPath, ps},
```

Comment:
Hmm this might require to have a

```
type PinPath struct{
  PinOptions
  Path string
}
```

in `api` and we use that type to wrap the information here.